app/presenters/social_relay_presenter.rb

```diff
@@ -0,0 +1,24 @@
+class SocialRelayPresenter
+
+  def as_json options={}
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `as_json(*)` if you want the method to accept any arguments but don't care about them.<br>Use def with parentheses when there are parameters.